plugins/controlboard/src/ControlBoardDriver.cpp

```diff
@@ -115,16 +135,11 @@ bool GazeboYarpControlBoardDriver::gazebo_init()
         }
         std::cout<<"INITIAL CONFIGURATION IS: "<<initial_config.toString()<<std::endl;
 
-        for (unsigned int i = 0; i < m_numberOfJoints; ++i) {
-#if GAZEBO_MAJOR_VERSION >= 4
-            m_jointPointers[i]->SetPosition(0,initial_config[i]);
-#else
-            gazebo::math::Angle a;
-            a.SetFromRadian(initial_config[i]);
-            m_jointPointers[i]->SetAngle(0,a);
-#endif
-        }
+        // Set initial reference
```

Comment:
This will be changing the behavior of the initial configuration tag from spawning the robot in the initial configuration, to just give the initial configuration as the initial desired position to the low level PID. This will result in a initial abrupt motion for models in wich the initial configuration is not the zero position (for example iCub arms). 


There is any reason for this change? @barbalberto